Userland/Libraries/LibAudio/FlacLoader.cpp

```diff
@@ -145,6 +148,38 @@ MaybeLoaderError FlacLoaderPlugin::parse_header()
     return {};
 }
 
+// 11.19. METADATA_BLOCK_PICTURE
```

Comment:
Note to self: update these references once the FLAC RFC is an actual IETF standard